research/picai/nnunet_scripts/nnunet_launch.slrm

```diff
@@ -35,16 +40,16 @@ BASE_DIR="$nnUNet_results/${DATASET_NAME}/nnUNetTrainer__${PLANS_IDENTIFIER}__${
 
 # Function returning array of args passed to nnUNetv2_train command
 get_nnunet_train_args() {
+	# $1 Refers to the first argument of this function and not first argument of script
 	local checkpoint_bool=$1
-	local arg_list = ( $DATASET_NAME $UNET_CONFIG $FOLD $PLANS_IDENTIFIER)
+	arg_list = ( $DATASET_NAME $UNET_CONFIG $FOLD $PLANS_IDENTIFIER)
```

Comment:
I know sometimes Bash is really annoying about white space. Will this work okay if there is no space after `$PLANS_IDENTIFIER`?